src/libs/antares/study/parts/thermal/cluster.cpp

```diff
@@ -608,6 +623,36 @@ bool Data::ThermalCluster::integrityCheck()
             ret = false;
     }*/
 
+    // ramping
+    if (maxUpwardPowerRampingRate <= 0)
+    {
+        logs.error() << "Thermal cluster: " << parentArea->name << '/' << pName
+                     << ": The maximum upward power ramping rate must greater than zero.";
+        maxUpwardPowerRampingRate = 1.;
+        ret = false;
+    }
+    if (maxDownwardPowerRampingRate <= 0)
+    {
+        logs.error() << "Thermal cluster: " << parentArea->name << '/' << pName
+                     << ": The maximum downward power ramping rate must greater than zero.";
+        maxDownwardPowerRampingRate = 1.;
+        ret = false;
+    }
+    if (powerIncreaseCost < 0)
+    {
+        logs.error() << "Thermal cluster: " << parentArea->name << '/' << pName
+                     << ": The ramping power increase cost must be positive or null.";
+        powerIncreaseCost = 0.;
+        ret = false;
+    }
+    if (powerDecreaseCost < 0)
+    {
+        logs.error() << "Thermal cluster: " << parentArea->name << '/' << pName
+                     << ": The ramping power decrease cost must be positive or null.";
+        powerDecreaseCost = 0.;
+        ret = false;
+    }
```

Comment:
```suggestion
    // ramping
    ramping.checkValidity(ret);
```

Reply:
Implemented in commit [b9801fe](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1817/commits/b9801feaf09d246c7f326859a24d4c2b97674d71)